conf/functions_worker.yml

```diff
@@ -45,6 +52,10 @@ metadataStoreCacheExpirySeconds: 300
 ################################
 # Function package management
 ################################
+packagesManagementStorageProvider:
+    org.apache.pulsar.packages.management.storage.bookkeeper.BookKeeperPackagesStorageProvider
+packagesManagementLedgerRootPath: /ledgers
+packagesReplicas: 1
```

Comment:
`broker.conf` and `standalone.conf` also have these same configurations.

If you added them here, what's the relationship between these duplicated configuration?

Do you want to remove them from other conf files or they will have some override relationship?

Reply:
Run Functions-worker separately, independent with broker, in this situation, if we want to start packageManagement service in worker, We should have this configurations.